payara/Payara
# FISH-7132 : handling deployment groups  in the same way done by clusters

## Description
In general, JMS Service availability doesn’t work when setting up a JMS cluster in LOCAL mode when using deployment groups unlike traditional clusters (generally known as Shoal clusters).

## Important Info
### Blockers
None

## Testing
### New tests
None

### Testing Performed
Steps:

1. build Payara Community
2. run it
3. Run the next commands using asadmin:
```
copy-config default-config mqc-config
set configs.config.mqc-config.availability-service.jms-availability.availability-enabled=true
set configs.config.mqc-config.jms-service.type=LOCAL

create-node-ssh --sshport=22 --nodehost=localhost --installdir=${com.sun.aas.productRoot} --sshuser=${user.name} remote-node-1
create-node-ssh --sshport=22 --nodehost=localhost --installdir=${com.sun.aas.productRoot} --sshuser=${user.name} remote-node-2

create-deployment-group dg1
create-instance --deploymentgroup=dg1 --node=remote-node-1 --config=mqc-config i1
create-instance --deploymentgroup=dg1 --node=remote-node-2 --config=mqc-config i2
start-deployment-group dg1

```
5. open server.log files for the 2 instances: i1 and i2
6. verify that it is exposing JMS Service in the 2 hosts like done in traditional clusters:
  ```
ADDRESSLIST in setJmsServiceProvider : mq://localhost:27677/,mq://localhost:27677/]]
  JMS Service Connection URL is : mq://localhost:27677/,mq://localhost:27677/]] 

```
### Testing Environment
Zulu JDK 11 on Windows 11 with Maven 3.8.4
